### PR TITLE
DM-29735: Add the DP0.1 schemas

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

**Review by gpdf — COMMENTED** [review]
Is it possible to follow the same "dp01_dc2" pattern in the Felis filename that we are using in the "schema" name?  In the long run that seems like it will be less confusing.

**Comment by hsinfang** [comment]
Sure. The file has been renamed in a new commit.

**Comment by gpdf** [comment]
It looks like the most recent commit, 8767950, added a "new" dc2_dp01.yaml, so its relationship to the previous content is broken ("git blame" attributes every line in the file to the last commit).

I would suggest rolling back to 7d1a218 in a clone, and then applying the changes needed to get from the 3bda983 to the 8767950 version of the file to the renamed file instead - and then force pushing.

**Comment by gpdf** [comment]
Just discussed with @hsinfang - I'll do the repair this evening.

**Comment by gpdf** [comment]
OK, fixed the file naming problem and amended commit message to make the point more visible.  Beware: **force-pushed.**

**Review by gpdf — COMMENTED** [review]
This looks good; we just need to add the agreed table descriptions.  You can also merge this and handle the descriptions as a separate PR if you wish.

For the record: I'll have to do more checks once this is actually deployed on TAP and I can see how it looks on the service.

Thanks!

File: yml/dp01_dc2.yaml [hunk 1]
```diff
@@ -1,6052 +1,7193 @@
...
-    description:
+    description: centroid shifted by more than the maximum allowed amount
+    fits:tunit: ''
   - name: y_base_SdssShape_flag_maxIter
-    "@id": "#dpdd_forced.y_base_SdssShape_flag_maxIter"
+    '@id': '#dpdd_forced.y_base_SdssShape_flag_maxIter'
     datatype: boolean
     mysql:datatype: BOOLEAN
-    description:
+    description: Too many ite
```
Comment (hsinfang) [inline yml/dp01_dc2.yaml]: why deleting this?

Comment (hsinfang) [inline yml/dp01_dc2.yaml]: why deleting this?

Comment (hsinfang) [inline yml/dp01_dc2.yaml]: Why deleting this?

**Comment by hsinfang** [comment]
It seems  https://github.com/lsst/sdm_schemas/pull/6/commits/3bda983aa5c2d21e6b015cd6b9d122f7256a70c3 got undone by https://github.com/lsst/sdm_schemas/pull/6/commits/32e7e96ac85bc4f20fffc7a15b06bda96a941436

**Comment by gpdf** [comment]
Yes, that was the repair I did.

**Comment by gpdf** [comment]
The history originally was:

- @hsinfang 7d1a218 renamed the file from dc2_dp01 to dp01_dc2
- @MelissaGraham 3bda983 added a "new" dc2_dp01, which broke the history with the old file
- @gpdf 32e7e96 used `git rebase -i` to replace Melissa's commit with a new one in which the changes were applied to dp01_dc2 and so preserved the file history

**Comment by hsinfang** [comment]
https://github.com/lsst/sdm_schemas/commit/3bda983aa5c2d21e6b015cd6b9d122f7256a70c3 is "Specify more non-nullable columns"   added by me before @MelissaGraham's commit.

**Comment by hsinfang** [comment]
@gpdf  I added the table name change and the table descriptions from @MelissaGraham 

Would you like to take a look before we merge this?

**Comment by gpdf** [comment]
I'm sorry, I got confused about what the old commit ID was.  I should have read your message more carefully.  Friday afternoon syndrome.  I admit I just assumed that Melissa had started with the version right before your renaming and - because there were so many changes just from the change from double quotes to single quotes - I didn't find that regression.

What is the story about the change of quotation marks, in any event?  Do we have two different scripts that are being used to generate Felis yaml from something else?

Anyway, thank you for catching this!

I think we are OK now unless you want to add the DESC version number as is being discussed on DM-29763.

**Comment by hsinfang** [comment]
Thanks. I'm merging this and will use this file to generate a TAP schema PR.  

The file has the table descriptions as @MelissaGraham  provided on the ticket and did not incorporate the newer comments such as adding version clarifications. If we want those it can be done in a new PR.

**Pull request merged**